pkg/services/featuremgmt/toggles_gen.json

```diff
@@ -155,6 +155,20 @@
         "hideFromDocs": true
       }
     },
+    {
+      "metadata": {
+        "name": "alertingImportYAMLUI",
+        "resourceVersion": "1747836716080",
+        "creationTimestamp": "2025-05-21T14:11:56Z"
+      },
+      "spec": {
+        "description": "Enables a UI feature for importing rules from a Prometheus file to Grafana-managed rules",
+        "stage": "GA",
```

Comment:
Shouldn't we use `experimental`?

Reply:
I think if we enable but default we cannot use experimental